### ISSUE TITLE
runs need to include network and contract address in their filenames

### ISSUE THREAD

**Issue by jessebmiller** [SquadGames/Aavenluutn#34]
If we want to maintain script history when moving to a new publisher contract without name collisions when referencing runs, we need to include the contract address and theoretically chain name/id?

/runs/0.0/json can refer to two different runs if the history is maintained across two different publisher contracts.

**Comment by EzraWeller** [comment]
Yup, though we could compensate by saying we'll only ever use one publisher at a time per chain and only use one chain per deployment
